ethereum/kiln/abis/0x858646372cc42e1a627fce94aa7a7033e7cf075a.abi.json

```diff
@@ -25,42 +25,28 @@
     {
         "inputs": [
             {
-                "internalType": "uint256[]",
-                "name": "strategyIndexes",
-                "type": "uint256[]"
-            },
-            {
-                "internalType": "address[]",
-                "name": "strategies",
-                "type": "address[]"
-            },
-            {
-                "internalType": "uint256[]",
-                "name": "shares",
-                "type": "uint256[]"
-            },
-            {
-                "internalType": "address",
-                "name": "withdrawer",
-                "type": "address"
-            },
+                "internalType": "bytes32",
+                "name": "",
+                "type": "bytes32"
+            }
+        ],
+        "name": "pendingWithdrawals",
+        "outputs": [
             {
                 "internalType": "bool",
-                "name": "undelegateIfPossible",
+                "name": "",
```

Comment:
No name expected here ? Is empty name ok ? 🤔

Reply:
yes no need to name outputs